pcs/cli/resource/parse_args.py

```diff
@@ -50,16 +50,15 @@ def parse_clone(arg_list, promotable=False):
         raise CmdLineInputError(
             "op settings must be changed on base resource, not the clone",
         )
-    parts["meta"] = prepare_options(
-        groups.get("options", []) + groups.get("meta", []),
-    )
+
+    if "meta" in groups:
+        raise CmdLineInputError(
+            "meta options is not supported in 'pcs resource clone' command"
```

Comment:
This function doesn't know it is called from `pcs resource clone` command. In fact, it is being called from `pcs resource clone`, `pcs resource promotable` and `pcs resource update`. The message should not refer to any specific pcs command.